src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/Extensions/SAP_Internal_Stats_KeyFigure+Formatting.swift

```diff
@@ -19,13 +19,28 @@ extension SAP_Internal_Stats_KeyFigure {
 				return nil
 			}
 
-			return String(format: AppStrings.Statistics.Card.million, formattedNumber)
+			formattedString = String(format: AppStrings.Statistics.Card.million, formattedNumber)
 		} else {
 			let decimals = max(0, Int(self.decimals))
 			numberFormatter.minimumFractionDigits = Int(decimals)
 			numberFormatter.maximumFractionDigits = Int(decimals)
 
-			return numberFormatter.string(from: NSNumber(value: value))
+			formattedString = numberFormatter.string(from: NSNumber(value: value))
```

Comment:
can we move the 10 mio stuff out to an extra method to make thing better readable?